Tools/Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py

```diff
@@ -31,15 +31,19 @@
 log = logging.getLogger('webkitscmpy')
 
 
-def _maybe_add_webkitcorepy_path():
-    # Hopefully we're beside webkitcorepy, otherwise webkitcorepy will need to be installed.
+def _maybe_add_library_path(library):
+    # Hopefully we're beside the provided library path, otherwise that library will need to be installed.
```

Comment:
I'd rather we explicitly checked for the case where we're in a site-packages directory, to make sure that explicitly fails.

Something like:

```python
def get_all_site_packages():
    rv = set()

    if not getattr(sys.flags, 'no_site', False):
        rv.update(Path(p).resolve() for p in site.getsitepackages())
        if site.ENABLE_USER_SITE:
            rv.add(Path(site.getusersitepackages()).resolve())

    return rv
    
file_path = Path(__file__).resolve()
if any(file_path.is_relative_to(p) for p in get_all_site_packages()):
    ...
```